addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -564,7 +564,7 @@
                             <label for="name" string="Product Name"/>
                             <h1><field name="name" placeholder="e.g. Lunch"/></h1>
                             <div name="options" groups="base.group_user" invisible="1">
-                                <div>
+                                <div class="d-inline-flex text-break">
```

Comment:
It seems that this part of code is never used. see "invisible="1" on the parent. I think we can remove this.